extensions/braze/actions/sendEmail/config/fields.ts

```diff
@@ -0,0 +1,33 @@
+import { type Field, FieldType } from '@awell-health/extensions-core'
+import { type ZodTypeAny, z } from 'zod'
+
+export const fields = {
+  appId: {
+    id: 'appId',
+    label: 'App Id',
+    description: 'App Identifier',
+    type: FieldType.STRING,
+    required: true,
+  },
+  from: {
+    id: 'from',
+    label: 'From Email',
+    description:
+      'Valid email address in the format "Display Name <email@address.com>"',
+    type: FieldType.STRING,
+    required: true,
+  },
```

Comment:
I'd rather split this up into two fields
- From email
- From name

The extension code merge the two together: `${fromName} <${fromEmail}>`